src/Components/LoadingAnimation/TableListLoading.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './TableListLoading.css';
+
+/**
+ * Empty table list loading component that shows up when user is waiting for the table list
+ */
+class TableListLoading extends React.Component {
+  constructor(props: any) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className="tableListSkeletons">
+        {[...Array(14)].map((value: undefined, index: number) => {
+          return (
+          <div className="tableSkeleton" key={index}>
+            <div className="tableTierLabelSkeleton"></div>
+            <div className="tableNameSkeleton"></div>
+          </div>
+          )
+        })}
```

Comment:
I think that last } should be maybe be on a new line?

Reply:
addressed